components/Schedule.tsx

```diff
@@ -30,48 +30,50 @@ const Schedule = ({
             ?.popularity ?? ClassPopularity.Unknown;
 
     return (
-        <Stack direction={"row"}>
-            {schedule.days.map((day) => (
-                <Box key={day.date} width={200} pr={2}>
-                    <Box py={2} width={200}>
-                        <Typography variant="h6" component="div">
-                            {day.dayName}
-                        </Typography>
-                        <Typography
-                            variant="h6"
-                            component="div"
-                            style={{
-                                color: theme.palette.grey[600],
-                                fontSize: 15,
-                            }}
-                        >
-                            {day.date}
-                        </Typography>
+        <Stack>
+            <Stack direction={"row"} margin={"auto"} spacing={2} px={1}>
```

Comment:
Do we need the extra `px={1}`? Consuming sparse space on mobile :upside_down_face: